addons/headless/script_component.hpp

```diff
@@ -17,3 +17,4 @@
 #include "\z\acex\addons\main\script_macros.hpp"
 
 #define DELAY_DEFAULT 15
+#define HC_BLACKLIST ["UAV"]
```

Comment:
maybe no need to hardcode it in macro. You can use GVAR  like in acemod/ACE3#5274

Reply:
`UAV` is the vehicle class type, but I think we want to prevent transfer of the AI "units" types

`"UAV_AI_base_F", "B_UAV_AI", "O_UAV_AI", "I_UAV_AI"`

Reply:
can't the check just be `unitIsUav` ?

Reply:
nope, `unitIsUav` is only for vehilces and not units :smile: 
```
typeOf z = "B_UAV_01_F"
unitIsUAV z = true
typeOf driver z = "B_UAV_AI"
unitIsUAV driver z = false
```